### PR TITLE
change management port to 7626

### PR DESCRIPTION
**Pull request by pjfanning**
see https://github.com/apache/incubator-pekko-management/issues/49

### CODE REVIEW

**Comment by mdedetrich** [comment]
@pjfanning Did you randomly pick the `6262` port or was it agreed upon somewhere?

**Comment by pjfanning** [comment]
> @pjfanning Did you randomly pick the `6262` port or was it agreed upon somewhere?

I made one up myself - 6262 in based on 'MANA' on a telephone keypad.

**Comment by mdedetrich** [comment]
> I made one up myself - 6262 in based on 'MANA' on a telephone keypad.

How about 6468 which is based on MGMT (i.e. management)? According to https://en.wikipedia.org/wiki/List_of_TCP_and_UDP_port_numbers its also free

**Comment by pjfanning** [comment]
Both 6262 and 6468 are clashes (Sybase and Discord, respectively)

I changed to 6458 inadvertently, I will choose another value though.

**Comment by pjfanning** [comment]
@mdedetrich how about 7626 for PMAN (Pekko MANagement)?

**Comment by mdedetrich** [comment]
I have an affinity for MGMT because of the music band but I am also happy with PMAN, in either case PR looks good.

You can ask some others if they have any preference but we shouldn't bike shed over this

File: management/src/main/resources/reference.conf [hunk 1]
```diff
@@ -16,7 +16,7 @@ pekko.management {
 
     # The port where the HTTP Server for Http Cluster Management will be bound.
     # The value will need to be from 0 to 65535.
-    port = 8558 # port pun, it "complements" 2552 which is often used for Pekko remoting
+    port = 7626 # PMAN (Pekko MANagement) on telephone keypad
```
Comment (nvollmar) [inline management/src/main/resources/reference.conf:19]: That would be ITU E.161 (https://en.wikipedia.org/wiki/E.161). There are programmers out there that never had phone with a keypad ;)

Comment (pjfanning) [reply management/src/main/resources/reference.conf]: extended the comment

**Comment by pjfanning** [comment]
@nvollmar @He-Pin @mdedetrich @jrudolph is this port ok?

**Review by nvollmar — APPROVED** [review]
lgtm

**Comment by He-Pin** [comment]
It's hard for me to choose a port, so leave to others, the overall pr looking good.

**Comment by WayneWang12** [comment]
`5885` might be a good choice

**Comment by WenjunHuang** [comment]
I nominate 6666. In Chinese, this number has a metaphor of smooth sailing and awesomeness.

**Comment by pjfanning** [comment]
> I nominate 6666. In Chinese, this number has a metaphor of smooth sailing and awesomeness.

This is problematic. Christians associate 666 with the devil. I don't think we can pick port numbers based on cultural terms, it's safer to avoid numbers that can be offensive to oversensitive people.

**Pull request merged**